<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileRecordSettingDialog</class>
 <widget class="QDialog" name="FileRecordSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>591</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>395</width>
    <height>591</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>File Record Config v0.4</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="npersonnelLabel">
       <property name="text">
        <string>Personnel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="npersonneLiineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nDeviceInfoLab">
       <property name="text">
        <string>DeviceInfo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nDeviceInfoLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="nChipInfoLab">
       <property name="text">
        <string>ChipInfo</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="nChipInfoLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="nRunNumLabel">
       <property name="text">
        <string>Run_num</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="nRunNumLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="nStartTimeLabel">
       <property name="text">
        <string>StartTime</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="nStartTimeLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="nEndTimeLabel">
       <property name="text">
        <string>EndTime</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="nEndTimeLineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="nLocationLabel">
       <property name="text">
        <string>Location</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="nLocationlineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="nDataTypeLabel">
       <property name="text">
        <string>DataType</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="nDataTypeComb"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="nGenomeSizeLab">
       <property name="text">
        <string>GenomeSize(bp)</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="nGenomeSizeLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="nPlasmidLab">
       <property name="text">
        <string>Plasmid(bp)</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="nPlasmidLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="nDnaLab">
       <property name="text">
        <string>LibraryDNA</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="nDnaLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="nTemperLabel">
       <property name="text">
        <string>Temperature</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="nTemperLineEdit"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="nBufferIdLab">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;BufferID&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="nBufferIdLineEdit"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="nPorinIdLab">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;PorinId&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="nPorinIdLineEdit"/>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="nMotorLab">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;MotorID&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLineEdit" name="nMotorLineEdit"/>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="nAdaptorLab">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;LibraryAdaptor&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLineEdit" name="nAdaptorLineEdit"/>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="nSampRateLab">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;OriginalSamplingRate(Hz)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="nSequenceLab">
       <property name="text">
        <string>SequencingVoltage(V)</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLineEdit" name="nSequenceLineEdit"/>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="nLowFilterLab">
       <property name="text">
        <string>LowPassFilter(Hz)</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QLineEdit" name="nLowFilterLineEdit"/>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="nFeatureLab">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Features&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QLineEdit" name="nFeatureLineEdit"/>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="otherInfoLab">
       <property name="text">
        <string>OtherInfo</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QLineEdit" name="OtherInfoLineEdit"/>
     </item>
     <item row="21" column="0">
      <widget class="QLabel" name="nFolderPathLab">
       <property name="text">
        <string>FolderPath</string>
       </property>
      </widget>
     </item>
     <item row="21" column="1">
      <widget class="QLineEdit" name="nFolderPathLineEdit"/>
     </item>
     <item row="16" column="1">
      <widget class="QSpinBox" name="nSampRatespbox">
       <property name="suffix">
        <string>K</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="nLoadTemplateBtn">
       <property name="text">
        <string>载入模板</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="nOkBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
